services/spar/src/Spar/API.hs

```diff
@@ -281,3 +286,22 @@ internalDeleteTeam :: TeamId -> Spar NoContent
 internalDeleteTeam team = do
   wrapMonadClient $ Data.deleteTeam team
   pure NoContent
+
+internalPutSsoSettings :: SsoSettings -> Spar NoContent
+internalPutSsoSettings SsoSettings{defaultSsoCode} = do
+  case defaultSsoCode of
+    Nothing -> do
+      wrapMonadClient $ Data.deleteDefaultSSOCode
+      pure NoContent
+
+    Just code -> do
+      wrapMonadClient (Data.getIdPConfig code) >>= \case
+        Nothing ->
```

Comment:
There are 5 levels of indentation in this function, making code harder to read as with fewer levels of indentation. Couldn't the first case split be done at function level (`internatlPutSsoSettings (SsoSettings Nothing) = ...`)?

Reply:
done, thanks.